### PR TITLE
Generate a layer with vehicle counts

### PR DESCRIPTION
**Pull request by dabreegster**
AADF (Number of vehicles that travel past the count point (in both directions) on an average day of the year), for https://github.com/acteng/atip/issues/318. Frontend PR in the works. Preview output at https://protomaps.github.io/PMTiles/?url=https%3A%2F%2Fatip.uk%2Flayers%2Fv1%2Fvehicle_counts.pmtiles

### CODE REVIEW

File: layers/traffic.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def makeDftVehicleCounts():
+    tmp = "tmp_vehicle_counts"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    run(
+        [
+            "wget",
+            # From https://roadtraffic.dft.gov.uk/downloads
```
Comment (dabreegster) [inline layers/traffic.py:12]: There are other options, like manual counts. AADF seems simplest to start with.

File: layers/traffic.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    run(["unzip", f"{tmp}/dft_traffic_counts_aadf.zip", "-d", tmp])
+
+    gj = {
+        "type": "FeatureCollection",
+        "features": [],
+    }
+
+    with open(f"{tmp}/dft_traffic_counts_aadf.csv") as f:
+        for row in csv.DictReader(f):
+            # Only keep the latest year
```
Comment (dabreegster) [inline layers/traffic.py:27]: Dunno what people will want to see; starting with something simple to visualize.

File: layers/traffic.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    "type": "Feature",
+                    "geometry": {
+                        "type": "Point",
+                        "coordinates": [float(row["Longitude"]), float(row["Latitude"])],
+                    },
+                    "properties": {
+                        "count_point": row["Count_point_id"],
+                        "location": f"{row['Road_name']} from {row['Start_junction_road_name']} to {row['End_junction_road_name']}",
+                        "method": row["Estimation_method_detailed"],
+                        "motor_vehicles_2022": int(row["All_motor_vehicles"]),
```
Comment (dabreegster) [inline layers/traffic.py:42]: They have a breakdown by vehicle type, but let's see what people want to see first. https://storage.googleapis.com/dft-statistics/road-traffic/all-traffic-data-metadata.pdf

File: layers/census.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import csv
```
Comment (Pete-Y-CS) [inline layers/census.py:1]: Is this in error? it looks unused

**Review by Pete-Y-CS — APPROVED** [review]
apart from seemingly erroneous import looks good

File: layers/census.py (hunk 4 above)
Comment (dabreegster) [reply layers/census.py:1]: No, I happened to notice the import was incorrectly in the main file and missing here. We do `csv.DictReader` below

**Pull request merged**